js/index.js

```diff
@@ -20,3 +20,49 @@ skills.forEach((skillName) => {
   skill.textContent = skillName;
   skillsList.appendChild(skill);
 });
+
+const messageForm = document.getElementsByName("leave_message")[0];
+// Add an event listener for the "submit" event
+messageForm.addEventListener("submit", function (event) {
+  event.preventDefault(); // Prevent the default form submission
+  const userName = event.target.usersName.value;
+  const userEmail = event.target.usersEmail.value;
+  const userMessage = event.target.usersMessage.value;
+  // Log the values to the console for verification
+  console.log("Name:", userName);
+  console.log("Email:", userEmail);
+  console.log("Message:", userMessage);
```

Comment:
IT's always better to remove `console.logs` after finishing implementation they used for debugging